go/systests/git_test.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2017 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package systests
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/keybase/client/go/git"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/client/go/teams"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGitTeamer(t *testing.T) {
+	tt := newTeamTester(t)
+	defer tt.cleanup()
+
+	alice := tt.addUser("abc")
+	g := tt.users[0].tc.G
+
+	teamer := git.NewTeamer(g)
+
+	t.Logf("team that doesn't exist")
+	res, err := teamer.LookupOrCreate(context.Background(), keybase1.Folder{
+		Name:       "/keybase/team/notateamxxx",
```

Comment:
@mlsteele I didn't catch this before:

I think folder `Name` fields should be of the form `notateamxxx` instead of `/keybase/team/notateamxxx`. See for example https://github.com/keybase/client/blob/master/go/client/cmd_favorite.go#L38-L56. Is it ok with you if I change this in my get/put PR?

Reply:
Fine by me. I forget where those Folder's come from. You'll probably want to do this by changing `ParseTeamPrivateKBFSPath ` to `keybase1.TeamNameFromString` [here](https://github.com/keybase/client/pull/8345/files#diff-6d49fd01d318b09fd5eeb6ae9cd5d2f4R51)

Reply:
LoadTeamArg.Name is just a string though. Any reason not to pass Folder.Name directly to it?

Reply:
Ah yeah, that's fine